build-system/server/lazy-build.js

```diff
@@ -25,6 +28,30 @@ const extensionBundles = {};
 maybeInitializeExtensions(extensionBundles, /* includeLatest */ true);
 
 /**
+ * Gets the unminified name of the bundle if it can be lazily built.
+ *
+ * @param {!Object} bundles
+ * @param {string} name
+ * @return {string}
+ */
+function maybeGetUnminifiedName(bundles, name) {
```

Comment:
That's not possible because when `argv.compiled` is `true`, it's still possible for `name` not to be present in `bundles` in either the minified or unminified form. In such cases, we fall through and simply return `name`.